docs/connectors/building.mdx

```diff
@@ -3,19 +3,89 @@ title: "Building Connectors"
 sidebar_position: 5
 ---
 
-Conduit connectors can be built in any programming language that supports gRPC.
-To make it easier to write connectors we provide
-a [Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk) written in
-Go. Using the SDK is the recommended way of writing a Conduit connector.
+## Developing Custom Connectors
 
-## Conduit connector template
+The Conduit platform offers extensibility through the development of connectors, which can be written in any programming language that supports [gRPC](https://grpc.io/). For ease of development, a Go-based [Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk) is provided and its use is the recommended approach for creating Conduit connectors.
 
-The easiest way to start implementing your own Conduit connector is by using the
-[Conduit connector template](https://github.com/ConduitIO/conduit-connector-template).
-It contains the basic project structure as well as some additional utilities
-like GitHub actions and a Makefile.
+## Initializing a Conduit Connector Project
+
+To begin the development of a custom Conduit connector, developers should initialize their project using the [Conduit connector template](https://github.com/ConduitIO/conduit-connector-template). This template streamlines the setup process by providing a fundamental project structure, along with utility configurations for GitHub actions and a Makefile.
+
+Included in the Conduit Connector Template are:
+
+- Base code for configuring the connector's source and destination.
+- Sample unit tests to validate connector functionality.
+- A preconfigured Makefile to assist with common build tasks.
+- GitHub Actions Workflows for continuous integration including building, testing, linting, and automated release creation upon tag push.
+- Dependabot configurations to automate dependency updates and minor version auto-merging.
+- Issue and Pull Request templates for consistent contribution practices.
+- A README template to guide project documentation.
+
+### Steps to Use the Connector Template
+
+1. On the repository's main page, select "Use this template".
+2. Provide the new repository details as prompted.
+3. Upon repository creation, clone it to your local development environment.
+4. Execute the `./setup.sh` script with the desired module name, e.g., `./setup.sh github.com/awesome-org/conduit-connector-file`.
```

Comment:
I'd add a callout here (e.g. an [admonition](https://docusaurus.io/docs/markdown-features/admonitions)), saying that by convention the name of the repository should be `conduit-connector-[connector name]`. So if you would like to [reference](https://conduit.io/docs/connectors/referencing) the connector using `foo`, the repository should be named `conduit-connector-foo`.

Reply:
@lovromazgon added note here: 69c01aa48ba24d6713e78d6578d4c60ca990fe19